client/src/app/tabs/PropertiesContainer.js

```diff
@@ -175,16 +262,43 @@ export default React.forwardRef(
 // helpers //////////
 
 function getLayout(dx, initialWidth) {
-  let width = Math.min(initialWidth - dx, MAX_WIDTH);
+  let width = initialWidth - dx;
+  const max_width = getMaxWidth();
 
-  const open = width >= MIN_WIDTH;
+  let open = width > MIN_WIDTH;
+  let fullWidth = width > max_width;
 
   if (!open) {
-    width = DEFAULT_LAYOUT.width;
+
+    // if was already closed and drags 40px
+    if (initialWidth < MIN_WIDTH && dx < -40) {
+
+      // snap to min_width
+      width = MIN_WIDTH;
+      open = true;
```

Comment:
I think we should add a test case here to verify it snaps to the min width. 🙂

Reply:
Added two more test cases for the min and max widths

Reply:
👍